src/Utils/normalization.jl

```diff
@@ -69,31 +61,72 @@ function normalize(system::AbstractSystem)
     throw(ArgumentError("the system type $(typeof(system)) is currently not supported"))
 end
 
-# systems already in canonical form, i.e. which don't need normalization
-normalize(system::CF) = system
+# x' = Ax, in the continuous case
+# x+ = Ax, in the discrete case
+function normalize(system::LCS{N, AN}) where {N, AN<:AbstractMatrix{N}}
+    n = statedim(system)
+    X = Universe(n)
+    return CLCS(system.A, X)
+end
+
+function normalize(system::LDS{N, AN}) where {N, AN<:AbstractMatrix{N}}
```

Comment:
This method can be merged with the one for `LCS` by following [this scheme](https://github.com/JuliaReach/MathematicalSystems.jl/blob/f65ac4c625b33069652f6b28169ae835dd021d84/src/systems.jl#L4-L15).